scripts/west_commands/runners/sctool.py

```diff
@@ -65,10 +65,13 @@ def _flash(self):
             exit()
         # get flash size
         flash_size = hex(build_conf['CONFIG_FLASH_SIZE'] * 1024)
+        flash_base = hex(build_conf['CONFIG_FLASH_BASE_ADDRESS'])
+        n22_partition_size = hex(build_conf['CONFIG_TELINK_W91_N22_PARTITION_SIZE'])
```

Comment:
Why do we have
flash_size = hex(build_conf['CONFIG_FLASH_SIZE'] * 1024)
and
n22_partition_size = hex(build_conf['CONFIG_TELINK_W91_N22_PARTITION_SIZE'])

Reply:
to use it for flashing tool without magic numbers
erase = subprocess.Popen(['./sctool', '-p', self.usb_port, '-da', 'da/da.ram.bin', '-b', '115200', '--manual', 'flash_erase', '0x80300000', '0x100000'], cwd=self.sctool_path)

Reply:
I think its not good when some partition size is provided in bytes another in kilobytes...

Reply:
fixed this